doc/ExpertOptions.md

```diff
@@ -1134,21 +1157,7 @@ In order to override the defaults for specific tasks, you can add the `perTaskDx
 
 will override the default timeout for tasks `Add` and `Inc`. It will also provide `UPLOAD` instead of `VIEW` project access to `Inc`.
 
-You are also able to specify metadata for tasks in the `defaultTaskDxAttributes` and `perTaskDxAttributes` sections, including adding citation or license information.
-
-The full set of attributes that may be specified are:
-
-* title
```

Comment:
I might be missing something, but where did this section go?  I don't see it on the right side of the diff.

Reply:
it is linked to the task meta section above. See line 1160 on the right.

Reply:
I don't see any mention of for example developerNotes at line 1160.
In fact, `git show 483d1523aa8e21f7a6553ac1693cdcfb36123367:doc/ExpertOptions.md` shows the usage of developerNotes in the global workflow section.
So were did the list of all the settable attributes go?

Reply:
in line 1160 I put a link pointing to https://github.com/dnanexus/dxCompiler/blob/fcbd5d1c1c376456e0715992b02cecd53d298d28/doc/ExpertOptions.md?plain=1#L566-L590
but there might be a caveat that in extras.json the keys are in camel case `developerNotes`, but in the meta section in wdl it is using underscore `developer_notes`.

Reply:
Great!  Please add this caveat and an example of developerNotes to the docs.